ci.cjs

```diff
@@ -0,0 +1,160 @@
+#! /usr/bin/env node
+
+const fs = require("fs").promises;
+const path = require("path");
+
+function printUsage() {
+  console.log(
+    `Usage: "ci check-consistency" or "ci bump-version %postfix%"`,
+  );
+}
+
+let postfix;
+const mode = process.argv[2];
+
+function validateArgs() {
+  switch (mode) {
+    case "get-version":
+      return true;
+
+    case "bump-version":
+      postfix = process.argv[3];
+      if (!postfix) {
+        printUsage();
+        process.exit();
+      }
+      return true;
+
+    case "":
+    case undefined:
+    case "check-consistency":
+      return true;
+
+    default:
+      return false;
```

Comment:
Maybe print `mode` with `console.err`, to report an error? and exit with 1, I guess